tools/checkpoint_conversion/test_keras_layer.py

```diff
@@ -0,0 +1,175 @@
+import keras
```

Comment:
Can we either delete this file out or move it into convert_qwen3_checkpoints (but actually use the in library `Qwen3LayerNorm`)?

As is this is just scratch work that is just named something really generic and doesn't even touch the library. We probably don't want to check it in.